Mathlib/Topology/Basic.lean

```diff
@@ -1362,6 +1362,20 @@ theorem ContinuousAt.eventually_mem {f : X → Y} {x : X} (hf : ContinuousAt f x
     (hs : s ∈ 𝓝 (f x)) : ∀ᶠ y in 𝓝 x, f y ∈ s :=
   hf hs
 
+/--
+Note: Version with (hl₁ : 𝓝 x ≤ l₁) is also true.
+
+Example usage:
+```
+example (x : ℝ) (f : ℝ → ℝ) (hf : Tendsto f (𝓝[>] x) atTop) : ¬ ContinuousAt f x := by
+  not_continuousAt_of_tendsto hf nhdsWithin_le_nhds (by simp)
+```
+-/
```

Comment:
I think just a simple one line description will do, like "If a function `f` tends to somewhere other than `𝓝 (f x)` at `x`, then `f` is not continuous at `x`".